screen-wake-lock/wakelock-onrelease.https.html

```diff
@@ -6,34 +6,17 @@
 <script src="/resources/testdriver-vendor.js"></script>
 <script>
 promise_test(async t => {
-  await test_driver.set_permission({name: 'screen-wake-lock'}, 'granted');
-
+  await test_driver.bless("screen wake lock");
   const lock = await navigator.wakeLock.request("screen");
-  return new Promise(resolve => {
+  const releaseEventPromise = new Promise(resolve => {
     lock.onrelease = resolve;
-    lock.release();
-  }).then(ev => {
-    assert_class_string(ev, "Event", "release() must fire an Event object");
-    assert_equals(ev.target, lock, "The event's target must be the lock that was acquired");
-    assert_true(ev.isTrusted);
-    assert_false(ev.bubbles);
-    assert_false(ev.cancelable);
   });
-}, "Test onreleased event's basic properties");
-
-promise_test(async t => {
-  await test_driver.set_permission({ name: 'screen-wake-lock' }, 'granted');
-
-  const lock = await navigator.wakeLock.request("screen");
-
-  let releaseFired = false;
-  lock.onrelease = t.step_func(() => {
-    releaseFired = true;
-  });
-
-  const releasePromise = lock.release();
-  assert_true(releaseFired, "The 'release' event fires immediately after release() is called");
-
-  return releasePromise;
-}, "Ensure onreleased is called before WakeLockSentinel.release() resolves");
```

Comment:
Why remove this test?

Reply:
I felt it was testing basically the same thing as the one above. Also, test itself doesn't prove that the the releasePromise resolved before the event fired.

I can write a test to actually check that though.

Reply:
Modified the test to check the order.